package.json

```diff
@@ -35,16 +35,21 @@
     "@atlaskit/field-text": "5.1.0",
     "@atlaskit/theme": "3.2.2",
     "electron-is-dev": "0.3.0",
+    "history": "4.7.2",
     "jitsi-meet-electron-utils": "jitsi/jitsi-meet-electron-utils",
     "react": "16.3.2",
     "react-dom": "16.3.2",
-    "react-router-dom": "4.2.2",
+    "react-redux": "5.0.7",
+    "react-router-redux": "5.0.0-alpha.9",
+    "redux": "4.0.0",
+    "redux-logger": "3.0.6",
     "styled-components": "3.3.0"
   },
   "devDependencies": {
     "babel-core": "6.26.3",
     "babel-eslint": "8.2.3",
     "babel-loader": "7.1.4",
+    "babel-plugin-transform-object-rest-spread": "6.26.0",
```

Comment:
Isn't this included in babel-preset-env ?

Reply:
I got an error. So I searched and found this. 
https://github.com/babel/babel-preset-env/issues/49

I guess they decide not to include this.

Reply:
I guess we can upgrade too. `@babel/preset-env`

https://github.com/babel/babel/tree/master/packages/babel-preset-env

Reply:
Let's go with `babel-preset-stage-1` since we also use it on jitsi-meet. It includes this plugin.

Reply:
Okay. I'll update this.